Formula/sha1dc.rb

```diff
@@ -0,0 +1,27 @@
+class Sha1dc < Formula
+  desc "Tool to detect SHA-1 collisions in files, including SHAttered"
+  homepage "https://github.com/cr-marcstevens/sha1collisiondetection"
+  url "https://github.com/cr-marcstevens/sha1collisiondetection.git",
+      :revision => "cb10f50d20a1c6c9efbf526c5da510c492d595b9"
+  version "HEAD-cb10f0"
+  head "https://github.com/cr-marcstevens/sha1collisiondetection.git"
+
+  depends_on "libtool" => :build
+
+  def install
+    inreplace "Makefile", "libtool", "glibtool"
+
+    # By default tries to build with HAVEAVX=1, fails.
+    inreplace "Makefile", "HAVEAVX=1", "HAVEAVX=0"
```

Comment:
Oh, duh. That works great for HAVEAVX (which itself is a workaround setting. Looks like https://github.com/cr-marcstevens/sha1collisiondetection/pull/4 tries to fix a bunch of this stuff more properly). LIBTOOL probably has to stay as a hack pending those kinds of fixes given that

```
LT_CC:=glibtool --tag=CC --mode=compile $(CC)
```
is probably too much to try to express on the command line, and there are pending changes to this area of the Makefile anyhow. (I didn't review open pull requests on the project before writing the recipe -- my bad)